doc/source/comparison_with_sas.rst

```diff
@@ -357,6 +357,142 @@ takes a list of columns to sort by.
    tips = tips.sort_values(['sex', 'total_bill'])
    tips.head()
 
+
+String Processing
+-----------------
+
+Length
+~~~~~~
+
+SAS determines the length of a character string with the ``LENGTHN`` 
+and ``LENGTHC`` functions. ``LENGTHN`` excludes trailing blanks and 
+``LENGTHC`` includes trailing blanks. 
+
+.. code-block:: none
+
+   data _null_;
+   set tips;
+   put(LENGTHN(time));
+   put(LENGTHC(time));
+   run;
+
+Python determines the length of a character string with the ``len`` function.
+``len`` includes trailing blanks.  Use ``len`` and ``rstrip`` to exclude 
+trailing blanks.
+
+.. code-block:: none
```

Comment:
Can you make all the python code segments ipython blocks (like elsewhere in the doc) so the output will show?